api/src/db/migrations/versions/2025_08_06_add_user_api_key_table.py

```diff
@@ -47,11 +47,15 @@ def upgrade():
     op.create_index(
         op.f("user_api_key_user_id_idx"), "user_api_key", ["user_id"], unique=False, schema="api"
     )
+    op.create_index(
+        op.f("user_api_key_key_id_idx"), "user_api_key", ["key_id"], unique=True, schema="api"
+    )
```

Comment:
Can't modify an existing migration once merged. While you asked if it has gotten to prod, as soon as you merged it, it went to dev/staging, so this change would not be applied there.